atlasdb-client/src/test/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueServiceTest.java

```diff
@@ -57,8 +57,8 @@
     private KeyValueService delegate;
     private KeyValueService kvs;
 
-    private static final Answer<Void> waitASecond = any -> {
-        Thread.sleep(1001);
```

Comment:
Sleep timer resolution, short of setting some extra JVM flags and doing some extra overhead we don't want to do, is pretty bad on Windows. I wrote a short test and sleep resolution in java was +- 2ms (which means 1001ms is sometimes 999ms, which flakes these tests) ; this matches what other people on the net normally see.

Even in linux, this wasn't _guaranteed_ to have millisecond resolution and was a flaky idea in general.